image/image.go

```diff
@@ -208,3 +209,51 @@ func createRuntimeBundle(w walker, dest, refName, rootfs string) error {
 
 	return json.NewEncoder(f).Encode(spec)
 }
+
+//Generate create an image that conforms to the OCI specification on a given path
+func Generate(dest string) error {
+	err := GenerateLayout(dest)
+	if err != nil {
+		return err
+	}
+
+	return Compress(dest)
+}
+
+//GenerateLayout create an imageLayou that conforms to the OCI specification on a given path
```

Comment:
“Layou” → “Layout”.  And do you have any plans to provide a way to generate a tar-backed image layout?  I [think we want to support that][1].

[1]: https://github.com/opencontainers/image-tools/pull/101#discussion_r93663062